src/main/kotlin/racingcar/ui/RaceGameController.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar.ui
+
+import racingcar.domain.Car
+import racingcar.domain.Cars
+import racingcar.domain.Race
+import racingcar.domain.RaceResult
+import racingcar.domain.strategy.RandomMovableStrategy
+
+object RaceGameController {
+    fun readyForRace(carNames: List<String>, totalRound: Int): Race {
+        val cars = Cars(carNames.map { Car(it) })
+
+        return Race(cars, totalRound)
+    }
+
+    fun runRace(race: Race): RaceResult {
+        return race.run(RandomMovableStrategy())
+    }
+
+    fun findWinner(raceResult: RaceResult): List<Car> {
+        return raceResult.getFinalRoundWinners()
+    }
```

Comment:
아래와 같이 변경해보면 어떨까요? 🤔
```suggestion
    fun play() {
        val race = readyForRace()
        val raceResult = race.run(RandomMovableStrategy())
        val winners = raceResult.getFinalRoundWinners()

        ResultView.printRaceResult(raceResult, winners)
    }

    private fun readyForRace(): Race {
        val names = InputView.getCarNames()
        val cars = Cars(names.map { Car(it) })

        val totalRound = InputView.getTotalRound()
        return Race(cars, totalRound)
    }
```

그리고 RaceGameApplication에서는 `RaceGameController.play()`과 같이 호출하면 될 것 같아요. 😉

[[10분 테코톡] 🧀 제리의 MVC 패턴](https://www.youtube.com/watch?v=ogaXW6KPc8I) 영상을 참고해보시면 좋을 것 같아요. 😃